x-pack/plugins/actions/server/lib/axios_utils.ts

```diff
@@ -57,7 +60,7 @@ export const request = async <T = unknown>({
     httpsAgent,
     proxy: false,
     maxContentLength,
-    timeout,
+    timeout: timeout != null ? timeout : settingsTimeout,
```

Comment:
feels werid to me that the hardcoded timeout would override the configured timeout, especially bc the configured timeout could be set longer. what about taking the longer timeout and using it?

Reply:
How do I set it? When I tried from `bedrock/bedrock.ts` I get an error that I cannot set configurationUtilities:
<img width="1092" alt="Screenshot 2023-10-05 at 3 57 18 PM" src="https://github.com/elastic/kibana/assets/6935300/62e51cbc-fee5-400c-aae6-3756574e607f">

Reply:
I am suggesting to compare the passed in `timeout` param (if defined) with the user configured `settingsTimeout` and use the longer interval for the axios timeout. Currently the `settingsTimeout` defaults to `60s` and I understand you want to set it longer for bedrock but it would be strange to me if a user (for whatever reason) configured the timeout to be `5m` and bedrock still used the hardcoded `2m` timeout. WDYT?

Reply:
How is a user able to configure the timeout? Could there be a reason to pass a shorter timeout that we aren't considering?

Reply:
It's a config available in `kibana.yml`: `xpack.actions.responseTimeout` https://www.elastic.co/guide/en/kibana/master/alert-action-settings-kb.html

I assume that we create this config to allow users to set the timeout longer than the default `60s`

Reply:
Gotchya. it seems weird that a lesser value would be ignored, but I don't care that much so implemented the change anyways and added a test. Let me know whatyou think